src/angular/components/register.component.ts

```diff
@@ -167,4 +167,11 @@ export class RegisterComponent {
         }
         return userInput;
     }
+
+    verifyPasswordHint() {
+        if (this.hint === this.masterPassword) {
+            this.platformUtilsService.showToast('error', 'Invalid password hint', 'Your password hint cannot be the same as your password.');
+            return;
+        }
```

Comment:
This should be moved up with all the other validation on submit.

Reply:
Updated the MR